Formula/kubesess.rb

```diff
@@ -19,6 +19,25 @@ class Kubesess < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+    bash_completion.install "./scripts/sh/completion.sh"
+    zsh_function.install "./scripts/sh/kubesess.sh"
+
+    fish_completion.install "./scripts/fish/completions/kc.fish"
+    fish_completion.install "./scripts/fish/completions/kn.fish"
+    fish_completion.install "./scripts/fish/completions/knd.fish"
+    fish_completion.install "./scripts/fish/completions/kcd.fish"
+
+    fish_function.install "./scripts/fish/functions/kc.fish"
+    fish_function.install "./scripts/fish/functions/kcd.fish"
+    fish_function.install "./scripts/fish/functions/kn.fish"
+    fish_function.install "./scripts/fish/functions/knd.fish"
```

Comment:
```suggestion
    bash_completion.install "scripts/sh/completion.sh"
    zsh_function.install "scripts/sh/kubesess.sh"

    %w[kc kn knd kcd].each do |basename|
      fish_completion.install "scripts/fish/completions/#{basename}.fish"
      fish_function.install "scripts/fish/functions/#{basename}.fish"
    end
```

Reply:
Added revison 1

Replaced my ugly code with your pretty one.

Reply:
Also fixed the commit message to suit the commit style guide, don't know how I forgot about that.